packages/flutter_tools/test/runner/flutter_command_runner_test.dart

```diff
@@ -39,11 +42,15 @@ void main() {
       fs.directory(_kProjectRoot).createSync(recursive: true);
       fs.currentDirectory = _kProjectRoot;
 
-      platform = FakePlatform(environment: <String, String>{
-        'FLUTTER_ROOT': _kFlutterRoot,
-      });
+      platform = FakePlatform(
+        environment: <String, String>{
+          'FLUTTER_ROOT': _kFlutterRoot,
+        },
+        version: '1 2 3 4 5',
```

Comment:
This addition isn't used

Reply:
It is used within the version.dart class.